api/v1alpha1/perses_types.go

```diff
@@ -38,7 +38,14 @@ type PersesSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	Affinity *corev1.Affinity `json:"affinity,omitempty"`
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +optional
+	// Image specifies the container image that should be used for the Perses deployment.
 	Image *string `json:"image,omitempty"`
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +optional
+	// serviceName specifies the name of the service that this perses instance will be associated with
+	// If not provided, the service name will be the same as the perses instance name
+	ServiceName *string `json:"serviceName,omitempty"`
```

Comment:
looking at the code, it seems that the empty string is not a correct value. So from my perspective you don't need to use a pointer of a string to evaluate if the user has set this value or not.

```suggestion
ServiceName string `json:"serviceName,omitempty"`
```

It will be simpler. Same comment for the `Image`